src/playdate/bindings/malloc.nim

```diff
@@ -0,0 +1,68 @@
+##
+## This file is a re-implementation of memalloc.nim in the Nim standard library. It is intended to be used by patching
+## it in as a replacement in your configs.nim file, like this:
+##
+## ```nim
+## patchFile("stdlib", "malloc", nimblePlaydatePath / "src/playdate/bindings/malloc")
```

Comment:
I action is required by game devs, I expect this patch to be applied to the sample project?

Reply:
No action should be required by game devs. This is setup to happen automatically if the project is properly configured.

Though I suppose I could add a clear error if the project _isn’t_ properly configured.

Reply:
I clarified the doc comments here to try and make this more clear.